src/components/Login/Login.css

```diff
@@ -0,0 +1,45 @@
+.button {
```

Comment:
buttons are common elements in web application. Consider moving this to App.css to make sure that all buttons looks similar

Reply:
.button is now defined in App.css and here. If button requires some special styling not defined in App.css do it here with id or different classname identifiers but remove overlapping styling.